src/init/inst-functions.sh

```diff
@@ -1117,6 +1117,18 @@ TransferShared()
     find ${INSTALLDIR}/etc/shared -maxdepth 1 -type f -not -name ar.conf -not -name files.yml -exec mv -f {} ${INSTALLDIR}/etc/shared/default \;
 }
 
+checkDownloadContent()
+{
+    VD_FILENAME='vd_1.0.0_vd_4.8.0.tar.xz'
```

Comment:
Again, the name of the file might change and we must find the best way to manage the file versioning.

Reply:
For the moment it will be handled like this.

Reply:
Ok, thank you!